src/modules/env_var.rs

```diff
@@ -253,6 +223,21 @@ mod test {
         assert_eq!(expected, actual);
     }
 
+    #[test]
+    fn display_single_with_multiple_defined() {
```

Comment:
We probably also want a test that tests this using the top-level `format` key. Is there any similar kind of test somewhere?